httpclient/metrics_round_tripper.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright © 2024 Acronis International GmbH.
+
+Released under MIT license.
+*/
+
+package httpclient
+
+import (
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+var (
+	ClientHTTPRequestDuration *prometheus.HistogramVec
```

Comment:
I suggest defining a separate struct `PrometheusMetrics` and having`HTTPRequestDuration` field within it. This would allow for easier extension in the future, which seems likely.

We already use this approach in other packages, for example:
+ https://github.com/acronis/go-appkit/blob/64b2af8edd00fc94e251c12fcfe6512f4ffa36d4/lrucache/metrics.go#L43
+ https://github.com/acronis/go-appkit/blob/64b2af8edd00fc94e251c12fcfe6512f4ffa36d4/httpserver/middleware/metrics.go#L72